ksql-cli/src/main/java/io/confluent/ksql/Ksql.java

```diff
@@ -35,6 +35,9 @@
 public class Ksql {
 
   public static void main(String[] args) throws IOException {
+    if (args.length == 0) {
```

Comment:
`final Options options = args.length == 0 ? Options.parse("http://localhost:8080") : Options.parse(args);`

Reply:
Question: Have we settled on `http://localhost:8080` as the default for the KSQL server endpoint?   Are 100% sure that it doesn't clash with any other CP components that might be running on the same machine when you do the CP quickstart?

Reply:
It does not have any problem working with other services in CP. But 8080 is kinda common port, it's default for tomcat, and may conflict with third party systems.

Reply:
@hjafarpour : Would you then recommend to change the KSQL default listener port from `8080` to something else, like `8090`?  Or keep it as `8080`?